src/bindings/python/fluxacct/accounting/test/test_job_archive_interface.py

```diff
@@ -261,7 +261,7 @@ def test_13_check_usage_factor_in_table(self):
         select_stmt = "SELECT usage_factor_period_0 FROM job_usage_factor_table WHERE username='1002' AND bank='C'"
         dataframe = pd.read_sql_query(select_stmt, acct_conn)
         usage_factor = dataframe.iloc[0]
-        self.assertEqual(usage_factor["usage_factor_period_0"], 17044.0)
+        self.assertEqual(usage_factor["usage_factor_period_0"], 16956.0)
```

Comment:
Do these test cases cover all three edges cases that are enumerated in the commit message? Or do we need to add more test cases to cover all three?

Reply:
Yes: 

- test 15 checks _Problem (1)_, where new jobs are found in a new half-life period.
- tests 11-13 checks _Problem (2)_, where new jobs are found in the current half-life period.
- test 17 checks _Problem (3)_, where no new jobs are found in a new half-life period or new jobs are found.